draft-ietf-httpbis-resumable-upload.md

```diff
@@ -48,7 +48,7 @@ normative:
   CACHING: RFC9111
   RFC9112:
     display: HTTP/1.1
-  STRUCTURED-FIELDS: RFC9651
+  STRUCTURED-FIELDS: RFC8941
```

Comment:
Is RFC 9651 not the correct reference for structured fields. As far as I understand it, RFC 8941 is obsoleted by RFC 9651.

Reply:
It is. I probably PRd based on a version too old. Please ignore this part.

Reply:
now fixed.